src/txmempool.h

```diff
@@ -724,6 +725,12 @@ class CTxMemPool
         return totalTxSize;
     }
 
+    CAmount GetTotalFee() const EXCLUSIVE_LOCKS_REQUIRED(cs)
+    {
+        AssertLockHeld(cs);
+        return m_total_fee;
```

Comment:
wouldn't this also work as a `std::atomic<CAmount>` and avoid the locking in the getter?

Reply:
Yes, but for the rpc to return atomic results, the lock needs to be taken. Also, it is already taken in the current code, so this is cheaper than an atomic.